pdf2text.js

```diff
@@ -0,0 +1,106 @@
+const fs = require("fs");
+const pdf = require("pdf-parse");
+
+const tmChars = "žäwertyuiopňöşasdfghjklzüçýbnm- ";
+
+console.log(process.argv);
+
+if (process.argv.length < 4) {
+  console.error("Arguments please");
```

Comment:
Can you change to more detailed log, like : Wrong number of arguments, expected 4